packages/maath/src/misc.ts

```diff
@@ -3,6 +3,53 @@ import { doThreePointsMakeARight } from "./triangle";
 import type { TypedArray } from "./ctypes";
 import { matrixSum3 } from "./matrix";
 
+/**
+ * Clamps a value between a range.
+ */
+export function clamp([min, max]: [number, number], value: number) {
+  return Math.max(min, Math.min(max, value));
+}
+
+/**
+ * Converts degrees to radians.
+ */
+export function degToRad(degrees: number) {
+  return (degrees / 180) * Math.PI;
+}
+
+/**
+ * Converts radians to degrees.
+ */
+export function radToDeg(radians: number) {
+  return (radians * 180) / Math.PI;
+}
+
+/**
+ * Gets 3D coordinates from latitude and longitude along a sphere.
+ *
+ * @param latitude Target latitude in degrees.
+ * @param longitude Target longitude in degrees.
+ * @param size Target sphere's dimensions in pixels.
+ * @param radius Target sphere's radius.
+ */
+export function getSphericalCoordinates(
+  latitude: number,
+  longitude: number,
+  size: { width: number; height: number },
+  radius: number
+) {
+  latitude = ((latitude - size.width) / size.width) * -180;
+  longitude = ((longitude - size.height) / size.height) * -90;
+
+  const convertedRadius = Math.cos(degToRad(longitude)) * radius;
+
+  const x = Math.cos(degToRad(latitude)) * convertedRadius;
+  const y = Math.sin(degToRad(longitude)) * radius;
+  const z = Math.sin(degToRad(latitude)) * convertedRadius;
+
+  return { x, y, z };
```

Comment:
We should decide wether to return vectors as {x,y,z} or a tuple [n, n, n]

Reply:
Yeah, should I cherry-pick this one into another PR until we sort out math classes?

Reply:
good idea, yep!